RELEASE.md

```diff
@@ -592,7 +606,7 @@ The 2.1 minor series tracks TensorFlow 2.1.
   - In the event of a transient backend issue or permanent breaking change, the
     uploader can now gracefully degrade and print a diagnostic (#2879)
 
-[TensorBoard.dev]: https://tensorboard.dev/
+[tensorboard.dev]: https://tensorboard.dev/
```

Comment:
I don't think this is correct. Maybe we should pull reformatting changes into a separate PR in the master branch, if you'd like? Let's just keep the new additions in this commit.